distributed/tests/test_joblib.py

```diff
@@ -99,7 +111,6 @@ def __reduce__(self):
         return (CountSerialized, (self.x,))
 
 
-@pytest.mark.parametrize('joblib', joblibs)
 def test_joblib_scatter(loop, joblib):
     if joblib is None:
         pytest.skip()
```

Comment:
We can probably remove these lines now here and in other tests.

Reply:
Oh, missed those. One sec.